src/spacel/aws/tag.py

```diff
@@ -0,0 +1,35 @@
+import logging
+
+logger = logging.getLogger('spacel')
+
+
+class TagWriter():
+    def __init__(self, clients, meta):
+        self._ec2 = clients.ec2()
+        self._instance_id = meta.instance_id
+
+    def update(self, manifest):
+        tags = manifest.tags
+        if not tags:
+            logger.debug('Tags not set.')
+            return
+
+        my_tags = []
+        for tag in tags:
+            prepared_tag = self._prepare(tag)
```

Comment:
Could this use key/values in a dict?
`{"foo":"bar", "foz":"baz"}` is easier/cleaner than `[{"key":"foo","value":"bar"},{"key":"foz","value":"baz"}]

You could get away with list comprehension for the whole kit: `my_tags = [{'Key':k, 'Value': v} for k, v in tags.items() if k and v]`?